client/src/components/themeSwitcher/ThemeSwitcher.jsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import classes from './themeSwitcher.module.css';
+import darkModeBtn from '../../../public/images/darkModeBtn.png';
+import lightModeBtn from '../../../public/images/lightModeBtn.png';
 
-function ThemeSwitcher() {
-  return <div></div>;
+function ThemeSwitcher({ theme, themeToggle }) {
+
+  const themeBtnImg = theme == 'light' ? darkModeBtn : lightModeBtn;
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.themeBtn} onClick={themeToggle}>
+        <img src={themeBtnImg}></img>
+      </div>
+      <span className={classes.themeText} data-theme={theme}>{theme}</span>
```

Comment:
What for do you need data-attribute here?

Reply:
to set a theme and load necessary color variable from style.css